visidata/clipboard.py

```diff
@@ -129,6 +129,9 @@ def delete_row(sheet, rowidx):
 
 @Sheet.api
 def paste_after(sheet, rowidx):
+    if not vd.memory.cliprows:  #1793
+        vd.warning('nothing to paste')
+        return
```

Comment:
Typically we use `vd.fail()` which is just a warning + exception.  This has the added benefit of carrying a stacktrace with the error that can be viewed with `Ctrl+E`.  Also, an exception will abort a replay, which is generally desirable with failed commands.

Reply:
Is aborting a replay and carrying a stacktrace desireable here? I am still developing my sense of taste around when to abort a replay and when to not. When the command is supposed to create rows or create columns, it is better to abort?

Reply:
I'm still developing my taste around that too =)

Let's try to empathize with the user here.  In this case, they have a replay script that selects some number of rows, and then they paste those rows in a different place or on another sheet, and then they go on to do something else with that data.

But in this dataset, for some legitimate reason it selects zero rows.  Should it refuse to proceed with the rest of the replay because of it?

Reply:
I would continue to replay. If this is intended by the user, everything is fine. If not intended, they will modify the input.

If we don't continue replay, the user has no choices

Reply:
The way I see it: if a replay fails due to bad input, that is on the user (with vd giving some useful troubleshooting output)